doc/how-to/getting_started_go.rst

```diff
@@ -1,256 +1,279 @@
-Manipulating the data
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. _getting_started-go-insert:
+.. _getting_started_go_import_:
 
-********************************************************************************
-Inserting data
-********************************************************************************
+Importing 'go-tarantool' packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To insert a :term:`tuple` into a :term:`space`, use ``Insert``:
+In the ``hello.go`` file, declare a ``main`` package and import the following packages:
 
-.. code-block:: go
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: package main
+    :end-before: func main()
+    :dedent:
 
-    resp, err = conn.Insert("tester", []interface{}{4, "ABBA", 1972})
+The packages for external MsgPack types, such as ``datetime``, ``decimal``, or ``uuid``, are required to parse these types in a response.
 
-This inserts the tuple ``(4, "ABBA", 1972)`` into a space named ``tester``.
 
-The response code and data are available in the
-`tarantool.Response <https://github.com/tarantool/go-tarantool#usage>`_
-structure:
+.. _getting_started_go_creating_connection:
 
-.. code-block:: go
+Connecting to the database
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    code := resp.Code
-    data := resp.Data
+1.  Declare the ``main()`` function:
 
-.. _getting_started-go-query:
+    ..  code-block:: go
 
-********************************************************************************
-Querying data
-********************************************************************************
+        func main() {
 
-To select a tuple from a space, use
-`Select <https://github.com/tarantool/go-tarantool#api-reference>`_:
+        }
 
-.. code-block:: go
+2.  Inside the ``main()`` function, add the following code:
 
-    resp, err = conn.Select("tester", "primary", 0, 1, tarantool.IterEq, []interface{}{4})
+    ..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+        :language: go
+        :start-after: func main()
+        :end-at: // Interacting with the database
+        :dedent:
 
-This selects a tuple by the primary key with ``offset = 0`` and ``limit = 1``
-from a space named ``tester`` (in our example, this is the index named ``primary``,
-based on the ``id`` field of each tuple).
+    This code establishes a connection to a running Tarantool instance on behalf of ``sampleuser``.
+    The ``conn`` object can be used to make CRUD requests and execute stored procedures.
 
-Next, select tuples by a secondary key.
 
-.. code-block:: go
 
-    resp, err = conn.Select("tester", "secondary", 0, 1, tarantool.IterEq, []interface{}{"ABBA"})
+.. _getting_started_go_using_data_operations:
 
-Finally, it would be nice to select all the tuples in a space. But there is no
-one-liner for this in Go; you would need a script like
-:ref:`this one <cookbook-select-all-go>`.
+Using data operations
```

Comment:
Maybe **Performing**?
Or just **Manipulating data**
**Using operations** sounds a bit strange to me